settings.py

```diff
@@ -880,10 +880,16 @@
 # using the local DEBUG value (which we don't have access to here yet)
 COMPRESS_ENABLED = lambda: not DEBUG and not UNIT_TESTING  # noqa: E731
 COMPRESS_OFFLINE = lambda: not DEBUG and not UNIT_TESTING  # noqa: E731
-COMPRESS_JS_COMPRESSOR = 'corehq.apps.hqwebapp.uglify.JsUglifySourcemapCompressor'
 # use 'compressor.js.JsCompressor' for faster local compressing (will get rid of source maps)
-COMPRESS_CSS_FILTERS = ['compressor.filters.css_default.CssAbsoluteFilter',
-'compressor.filters.cssmin.rCSSMinFilter']
+COMPRESS_FILTERS = {
+    'js': [
+        'corehq.apps.hqwebapp.uglify.JsUglifySourcemapCompressor',
```

Comment:
This file is not currently compatible with this version of django-compressor, but thankfully hasn't been used in many years. Production currently relies on the default js compressor. This commit still has merit, however, in that it moves `COMPRESS_CSS_FILTERS`, which is deprecated, to the `css` key here. If you want to re-do this, I'd keep that part, remove the `COMPRESS_JS_COMPRESSOR` variable entirely, and leave the `js` key unset (assuming that will give us the default compressor)

Reply:
Just a heads up, I created https://github.com/dimagi/commcare-hq/pull/29338 to handle removing the setting and module